templates/common-params.tpl

```diff
@@ -8,8 +8,7 @@
   type: string
   default: ca-bundle.crt
   description: |
-    Certificate Authority (CA) bundle filename on the `ssl-ca-directory`
-    Workspace.
+    Certificate Authority (CA) bundle filename in the SSL CA directory.
```

Comment:
Should we add `Workspace` to indicate its a workspace type ?
```suggestion
    Certificate Authority (CA) bundle filename in the SSL CA directory Workspace.
```

Reply:
No, because this parameter also goes into the StepAction and StepActions don't support workspaces. If the workspace wording needs to be in the task, then can we somehow move this parameter out of common parameters and into task and StepAction specific parameters, so the descriptions can be differfent?

Reply:
Oh ya make sense 
i think we can consider this as it is